src/bridge/IrcBridge.ts

```diff
@@ -685,9 +717,12 @@ export class IrcBridge {
     private async _onEvent (baseRequest: Request): Promise<BridgeRequestErr|undefined> {
         const event = baseRequest.getData();
         let updatePromise: Promise<void>|null = null;
-        if (event.sender && this.activityTracker) {
+        if (event.sender && (this.activityTracker ||
+            this.config.ircService.metrics.userActivityThresholdHours !== undefined)) {
```

Comment:
* `ircService` may not be defined according to the schema.
* `metrics` may not be defined according to the schema.

Reply:
>ircService may not be defined according to the schema.

That's bad, it definitely should be defined.

> metrics may not be defined according to the schema.

That sound possible, our types are wrong.